src/views/catalog/CreateFromInstanceTypes/CreateFromInstanceType.tsx

```diff
@@ -56,6 +57,14 @@ const CreateFromInstanceType: FC = () => {
 
   const namespace = ns === ALL_NAMESPACES_SESSION_KEY ? DEFAULT_NAMESPACE : ns;
 
+  const hasNameAndInstanceType = useMemo(
```

Comment:
why did u choose to calc hasNameAndInstanceType here and not in CreateVMFooter ? 
CreateVMFooter already get all those props of vmName, selectedbootablevolume and  , selectedInstanceType ?
why add more props instead of getting the needed value calc in there final location where its needed..

Reply:
The answer is right in front of you, see [this code change](https://github.com/kubevirt-ui/kubevirt-plugin/pull/1158/files/84b9f37427132340b2c28af146f1117cbdb66527#diff-b3e791410ef7aa2ba5068a197aaa56b96553c0c8207d308ed31ce7962cfef9bfR64) :)
To explain more: info about selected InstanceType was missing in `CreateVMFooter`.
So I had 2 options: to make the required calculations in `CreateVMFooter` and add another arg representing selected InstanceType to be able to do those calcs in there, or to make it one component above, where all the info I need is, and just to pass already calculated data to `CreateVMFooter`.
What does it mean? It means we'd have to add another arg to `CreateVMFooter` anyway.

Reply:
well im guessing that from the vm u can get the missing info @avivtur which fields those she needs?

Reply:
Hi :) 
as I can see in according to [this](https://github.com/kubevirt-ui/kubevirt-plugin/pull/1158/files#diff-b3e791410ef7aa2ba5068a197aaa56b96553c0c8207d308ed31ce7962cfef9bfR146-R152)
the generated VM should have the required fields to check under the VM in the footer component.
vm.spec.instancetype.name should hold the name for the IT
and as vm name I think is quiet clear how to get this :)

Reply:
I think I need only InstanceType name to get properly from the `vm`, in my case. Thanks so much for your help, Aviv and Matan! ❇️